ResearchKit/Common/ORKImageCaptureView.m

```diff
@@ -156,35 +149,57 @@ - (void)setError:(NSError * __nullable)error {
 const CGFloat CONTINUE_ALPHA_OPAQUE = 0;
 
 - (void)updateConstraints {
+    if (_variableConstraints) {
```

Comment:
My understanding of our reading of the documentation is that removal of old constraints should happen before setNeedsUpdateConstraints, rather than in the updateConstraints method. Is that not your understanding?